src/app/core/utils/functions.ts

```diff
@@ -82,3 +82,8 @@ export function decamelizeString(str: string): string {
         .join(' ')
     : str;
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function isPromise(obj: any): obj is Promise<any> {
```

Comment:
this function is not necessary and can be removed - use isPromise function from node util types

Reply:
utils (CommonJs library) is not per default included in webpack 5 and is not recommended to use with Angular (https://angular.io/guide/build#configuring-commonjs-dependencies)